src/shared/components/events-timeline-title/index.js

```diff
@@ -11,12 +11,12 @@ const EventsTimelineTitle = ({
     case 'today':
       return (<h2>Today</h2>);
     default:
-      return null;
+      return (<noscript />);
```

Comment:
we have to return `<noscript />` instead of null since React 0.14 complains otherwise. This is fixed in the current React version.